### DISCUSSION TITLE
NextJS + Prisma + SQLite: production does not show modified data until rebuild

### DISCUSSION THREAD

**Discussion by berenteb** [prisma/prisma#17519] [Q&A] [answered]
In development mode, everything happens just in time, but when I build the whole application and run it, modifications made via the app are saved according to sqlite. However, on page reload, the modified (deleted, edited, created) data does not show until I rebuild the app and run again.

I have tried to disable cache-control to no-store, but no luck, I also tried to move around the database, but nothing.

Do I miss sth here?

**Comment by nurul3101** [comment, marked as answer]
Hey @berenteb 👋 

Does the SQLite database show the updated data when changes are made through the deployed app?
If yes, that would mean that there is some issue in data fetching.

You can also check if your SQLite database file is being locked by another process. This can cause problems if you're trying to access it from multiple processes at the same time.

**Reply by berenteb** [reply]
Thanks for your reply. After I followed the path of how Next caching the data (I logged out every query), I have found out that caching is disabled in development mode and should be configured in production mode. I have set revalidation to 0 (which seems to mean every time), therefore every fetch generates a new page. Not ideal usage, but gets the job done.

**Reply by nurul3101** [reply]
Thanks for sharing the solution that worked for you! 👏

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
